docroot/themes/custom/mass_theme/templates/block/block--mainnavigation.html.twig

```diff
@@ -31,67 +31,18 @@
  * http://www.w3.org/TR/WCAG-TECHS/H42.html for more information.
  */
 #}
-{% set ariaLabel = heading_id ?? 'main-navigation-heading' %}
-<nav class="ma__header__nav" role="navigation" aria-labelledby="{{ ariaLabel }}" id="main-navigation">
-
-
-  {# Label. If not displayed, we still provide it for screen readers. #}
-  {% if not configuration.label_display %}
-    {% set title_attributes = title_attributes.addClass('visually-hidden') %}
-  {% endif %}
-  {{ title_prefix }}
-  <h2{{ title_attributes.setAttribute('id', ariaLabel) }}>{{ configuration.label }}</h2>
-  {{ title_suffix }}
-
-  <div class="ma__header__button-container js-sticky-header">
-    <button type="button" class="ma__header__back-button js-close-sub-nav"><span>Back</span></button>
-    <button type="button"
-            aria-expanded="false"
-            aria-label="Open the main menu for mass.gov"
-            class="ma__header__menu-button js-header-menu-button">
-      <span>Menu</span><span class="ma__header__menu-icon"></span>
-
-      {# Menu label with screen size greater than 620px. #}
-      {# <span class="ma__header__hamburger__menu-text ma__header__hamburger__desktop-label js-header__menu-text">Menu</span> #}
-      {# Menu label with screen size less than 620px. #}
-      {# <span class="ma__header__hamburger__menu-text ma__header__hamburger__mobile-label js-header__menu-text">Mass.gov</span> #}
-
-    </button>
-
-    {# Skip button to search. #}
-    {# <button type="button"
-            aria-expanded="false"
-            class="ma__header__hamburger__search-access-button js-header-search-access-button">
-      <span class="ma__visually-hidden">Access to search</span>
-      {{ icon('search') }}
-    </button> #}
-
-  </div>
-
-  {# 2 NARROW UTILITY NAVS? #}
-  <div class="ma__header__utility-nav ma__header__utility-nav--narrow js-utility-nav--narrow">
-    {{ render_menu('utility') }}
-  </div>
-  {#
-  <div class="ma__header__hamburger__utility-nav ma__header__hamburger__utility-nav--wide js-utility-nav--wide">
-    {{ render_menu('utility') }}
-  </div>
-  #}
-
+{# This is the horizontal navigation component.
+   In Home page, this block is placed after
+   themes/custom/mass_theme/templates/block/block--mainnavigation-2.html.twig.
+   It should only contains the horizontal main navigation, no utlity nav, no search, no logo.
+#}
+<nav class="ma__header__nav" role="navigation" id="main-navigation"  aria-label="Main navigation">
   <div class="ma__header__nav-container">
-    <div class="ma__header__nav-search">
```

Comment:
Is it because that this version is only used on homepage so it no longer needs the searchbar in the header area?

Reply:
This is a part of the old mobile version which used to fly out from the right side.  Since the new mobile version is not using this part as it's in `div.ma__header__hamburger__nav-container`, this can be removed.